monai/bundle/workflows.py

```diff
@@ -129,6 +130,23 @@ def get_workflow_type(self):
         """
         return self.workflow
 
+    def add_property(self, name: str, required: str, desc: str | None = None):
+        """
+        Besides the default predefined properties, some 3rd party aplications may need the bundle
+        dfinition to provide additonal properties for the specific use cases, if the bundlle can't
+        provide the property, means it can't work with the application.
+        This utility adds the property for the application requirements check and access.
+
+        Args:
+            name: the name of target property.
+            required: whether the property is "must-have".
+            desc: description document for the property.
+
+        """
+        if name in self.properties:
+            warnings.warn(f"property '{name}' already exists in the properties list, overriding it.")
```

Comment:
> overriding it

suggest to say:
```
please overriding it.
```

Reply:
No, this one is not asking user to override it.
Just let users know we will override it in the next line code.

Thanks.